enhancements/builds/entitlements.md

```diff
@@ -0,0 +1,295 @@
+>that was the case in 3.x.
+
+## Summary
+
+So in V3.x, all OpenShift Builds essentially had the customer RHEL entitlements mounted into the build pod automatically
+and without further intervention after install "for free" because:
+
+ - the ansible based installer required/forced providing your subscription information in order to install OCP
+ - for reference, see [these docs](https://docs.openshift.com/container-platform/3.11/install/host_preparation.html#host-registration) 
+ and [these roles](https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/rhel_subscribe/tasks/main.yml)
+ - next, Red Hats's docker daemon installed in 3.x had specific logic that auto-mounted the host's subscriptions 
+ into the container
+
+In V4.x, with 
+
+- the removal of the docker daemon
+- and the total replacement of the openshift ansible based installer
+
+all those 3.x mechanisms are gone, and a series of steps from either the cluster admin and/or developer attempting
+to run OpenShift Builds leveraging entitled content must be performed after install to enable access to the entitled
+content during the build.
+
+The highlights of those steps from [the current 4.x doc](https://docs.openshift.com/container-platform/4.3/builds/running-entitled-builds.html):
+
+1) First, either
+ 
+- create an ImageStreamTag to the UBI image from `registry.redhat.io` in the `openshift` namespace, or 
+- reference the UBI image directly and include a pull secret to `registry.redhat.io` in your BuildConfig
+
+2) Next, based on your access to the credentials:
+
+- With direct access to the entitlement pem files, create a generic secret from the files and reference the secret in
+your BuildConfig's input/source secret list
+- With Subscription Manager, and S2I build strategy, create config maps for the Subscription Manager configuration and 
+certs and reference the config maps in your BuildConfig's input/source config map list
+- With Subscription Manager, and Docker build strategy, you have to modify your Dockerfile and manually copy and 
+initialize as needed the information from the base image
+- With Satellite, and S2I build strategy, create a config map to reference the repository config file you created for your
+satellite instance and reference the config map in you BuildConfig's input/source config map list
+- With Satellite, and Docker build strategy, you have to modify your Dockerfile and manually copy and initialize as 
+needed the information from the base image as well as your repository config file for your satellite instance
+- And remember to set the skip layers optimization option (i.e. squash layers) with Docker Strategy builds
+
+This enhancement proposal aims to introduce more centralized and automated infrastructure to ease the burden of 
+using OpenShift Builds with RHEL subscriptions/entitlements.
+
+## Motivation
+
+Simplify overall usability for using RHEL Subscriptions/Entitlements with OpenShift Builds, as we took a hit in 
```

Comment:
i would like to add a motivation that ensures we keeps this process simple for an administrator that manages larger numbers of clusters.

Reply:
maybe should be listed as a use case as well ("as an administrator of a large number of clusters I want an easy way to enable builds w/ subscription content on all of those clusters")

(note that each cluster may still need unique creds, depending on the api rate limiting behavior)

Reply:
I've added the user stories based on comments in that section.  An update to the motivation section will be part of the next push.